torch/_inductor/scheduler.py

```diff
@@ -1624,6 +1629,9 @@ def score_fusion_memory(self, node1, node2):
         common_memory_deps = (node1.read_writes.reads | node1.read_writes.writes) & (
             node2.read_writes.reads | node2.read_writes.writes
         )
+        common_memory_deps = {
+            dep for dep in common_memory_deps if not dep.has_unbacked_symbols()
```

Comment:
For this one, wouldn't you prefer to use a fallback heuristic number?

Reply:
I'm somewhat hesitant to use a heuristic number here: if the number is too large, this may lead to overoptimistic saving estimation; if too small, this likely won't make a substantial difference. So my thinking was to (somewhat conservatively) drop the dependencies with unbacked SymInts in them, as no reasonably close estimation can be done (like with backed SymInts). Does this make sense?

Reply:
ok